src/modules/fundingManager/token/interfaces/IERC20Issuance_blacklist_v1.sol

```diff
@@ -1,6 +1,10 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity 0.8.23;
 
+//--------------------------------------------------------------------------
```

Comment:
```suggestion
```

Reply:
I think I copy pasted the the lines here, which shouldn't have been. Apologies for the confusion